src/validators/BirthdateInputValidator.js

```diff
@@ -50,34 +41,26 @@ export const getMaxDateValidator = (props) => {
 }
 
 export const getMinDateValidator = (props) => {
-  const {
-    minDate,
-    customErrorMessage,
-    dateFormat,
-  } = props
+  const { minDate, customErrorMessage, dateFormat } = props
 
   return (value) => {
     // Should pass if there is no value
     if (value == null || value === '') return ''
-    const date = dayjs(minDate);
-    const dayjsFormat = dateFormat.toUpperCase();
+    const date = dayjs(minDate)
+    const dayjsFormat = dateFormat.toUpperCase()
     const errorMessage =
       customErrorMessage ||
       `Please enter a date before ${date.format(dayjsFormat)}`
     return dayjs(value).isAfter(date) ? '' : errorMessage
   }
 }
 
-export const getMinMaxDateValidator = (props) => {
-  const {
-    minAge,
-    maxAge,
-    dateFormat,
-    customErrorMessage,
-  } = props;
+export const validateMinMaxDateFactory = (props) => {
```

Comment:
Massive change here from prettier, the only thing i actually changed in this file was the name of this exported validator.